### ISSUE TITLE
Error getting request for apiKey: DESCRIBE_CONFIGS

### ISSUE THREAD

**Issue by andrejvanderzee** [Mongey/terraform-provider-kafka#46]
With TF `v0.11.11` and Kafka `1.0.2` we get the following error after successfully creating a topic:

```Feb 28 15:47:43 ip-10-20-0-4 docker[13089]: org.apache.kafka.common.errors.InvalidRequestException: Error getting request for apiKey: DESCRIBE_CONFIGS, apiVersion: 1, connectionId: 10.20.0.4:9093-172.19.202.139:49218-4, listenerName: ListenerName(SSL), principal: User:CN=provision,OU=SRE,O=XXX,L=Amsterdam,ST=Noord-Holland,C=NL
Feb 28 15:47:43 ip-10-20-0-4 docker[13089]: org.apache.kafka.common.errors.InvalidRequestException: Error getting request for apiKey: DESCRIBE_CONFIGS, apiVersion: 1, connectionId: 10.20.0.4:9093-172.19.202.139:49218-4, listenerName: ListenerName(SSL), principal: User:CN=provision,OU=SRE,O=XXX,L=Amsterdam,ST=Noord-Holland,C=NL
```

**Comment by andrejvanderzee** [comment]
Compiled master from source and now works as expected.

**Comment by Mongey** [comment]
Thanks, I believe his was fixed in #45 . I'll release a new version with the fix.
